policyengine_us/variables/gov/states/ri/tax/income/agi/subtractions/taxable_retirement_income/ri_taxable_retirement_income_subtraction_eligible.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class ri_taxable_retirement_income_subtraction_eligible(Variable):
```

Comment:
only taxable elements can be subtracted so we can delete `taxable` from these variables and parameters
```suggestion
class ri_retirement_income_subtraction_eligible(Variable):
```